### ISSUE TITLE
Privacy Policy and T&C on Download workflow

### ISSUE THREAD

**Issue by dvenprasad** [AlexsLemonade/refinebio#265]
### Context
The user is required to agree to the Privacy Policy and Terms and Conditions before they can download data from refine.bio. This step needs to be incorporated as part of the download workflow.

### Problem or idea
Add an additional step in the download workflow to have the user agree to the privacy policy and terms and conditions at least once. We can use cookies to store whether or not they've agreed to the terms or not. 

### Solution or next step
Address this issue in the first sprint after keytar kurt.

**Comment by cgreene** [comment]
I just wanted to note that this also applies to downloads via the API. We will also need to give people some form of API key when they click a checkbox nothing their agreement.

**Comment by dvenprasad** [comment]
The user will need to agree to T&C at two of the download states. 

![download-dataset-processing](https://user-images.githubusercontent.com/15315514/41618457-391576ca-73d1-11e8-977b-509e70157095.png)


![download-dataset-download-ready](https://user-images.githubusercontent.com/15315514/41618472-4491567c-73d1-11e8-8887-7be1caf3aee1.png)

@cgreene Will this satisfy the legal requirements?

@ramenhog Invision links:
- [https://invis.io/TRLBRQIVNSZ#/292620200_download-Dataset-Processing](https://invis.io/TRLBRQIVNSZ#/292620200_download-Dataset-Processing)
- [https://invis.io/3HLBRMY7RDU#/302048787_download-Dataset-Download-Ready](https://invis.io/3HLBRMY7RDU#/302048787_download-Dataset-Download-Ready)

Illustrations are in the Assets folder

**Comment by Miserlou** [comment]
This still doesn't seem right - email and checkbox should be confirmed
before the smash is dispatched/emailed.

**Comment by dvenprasad** [comment]
@cgreene 
Users will only need to confirm via checkbox only once, unless they clear their cookies. The two mockups are for two different use cases. 

The first mockup is what is displayed when the user first initiates the download. They need to agree to terms here because we will them an email with a link to the download. 
Now, if the same user re-visits the URL, and the download is ready, they will just see an active download button (#305)

Say the user shares the link with someone, and when the person they've shared it with visits the page and the download is ready, then they will see the second mockup (if they've never downloaded data from refine.bio) 

There is a separate ticket (#291) for the API token generation. 

@Miserlou That sounds like that's a separate issue.
